Formula/gst-plugins-bad.rb

```diff
@@ -40,6 +40,10 @@ class GstPluginsBad < Formula
   depends_on "srtp" => :optional
   depends_on "libvo-aacenc" => :optional
 
+  # gst-libs/gst/gl/cocoa/gstglwindow_cocoa.h is missing in the 1.12.1 release. Fixed by this commit:
+  # https://cgit.freedesktop.org/gstreamer/gst-plugins-bad/commit/?id=450b1c92abe2ae8f84e557f61c9512a1f4006bab
+  patch :DATA
```

Comment:
With the file added as in this patch or do you prefer to have the upstream patch applied? I first tried that but then I had issues with invoking automake during the build.